src/libraries/System.Private.CoreLib/src/System/Numerics/Quaternion.cs

```diff
@@ -8,28 +8,32 @@
 namespace System.Numerics
 {
     /// <summary>Represents a vector that is used to encode three-dimensional physical rotations.</summary>
+    /// <remarks>The <see cref="System.Numerics.Quaternion" /> structure is used to efficiently rotate an object about the (x,y,z) vector by the angle theta, where:
+    /// ```
```

Comment:
Not sure about using \``` in XML. Also, it should have a code language.

Reply:
Good catch. These are the cases that we need to manually move to a file in dotnet-api-docs, then link to it here.

Reply:
Found a bunch of these and I have converted them either to remarks with format + cdata, or to `<c></c>`.